swagger2.0/oic.r.softwareupdate.swagger.json

```diff
@@ -0,0 +1,199 @@
+                  "result" : 0,
+                  "lastupdate" : "2015-01-09T14:30Z",
+                  "signage" : "vendor",
+                  "updatetime" : "2030-01-09T14:30Z"
+                },
+              "schema": { "$ref": "#/definitions/swupdate" }
+            }
+        }
+      }
+    }
+  },
+  "parameters": {
+    "interface-rw" : {
+      "in" : "query",
+      "name" : "if",
+      "type" : "string",
+      "enum" : ["oic.if.rw"]
+    },
+    "interface-all" : {
+      "in" : "query",
+      "name" : "if",
+      "type" : "string",
+      "enum" : ["oic.if.rw", "oic.if.baseline"]
+    }
+  },
+  "definitions": {
+    "swupdate" : {
+      "properties": {
+        "rt" : {
+          "items": {
+            "enum": [
+              "oic.r.softwareupdate"
+            ],
+            "type": "string"
+          },
+          "maxItems": 2,
+          "minItems": 2,
+          "type": "array",
+          "readOnly": true,
+          "uniqueItems": true
+        },
+        "if": {
+          "description": "The interface set supported by this resource",
+          "items": {
+            "enum": [
+              "oic.if.baseline",
+              "oic.if.rw"
+            ],
+            "type": "string"
+          },
+          "minItems": 2,
+          "maxItems": 2,
+          "type": "array",
+          "readOnly": true,
+          "uniqueItems": true
+        },
+        "n": {
+          "$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.common.properties.core-schema.json#/definitions/n"
+        },
+        "id": {
+          "$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.common.properties.core-schema.json#/definitions/id"
+        },
+        "purl" : {
+          "description": "Source of the software package, might be a HTTPS or CoAPs URL",
+          "maxLength": 64,
+          "type": "string",
+          "format": "uri"
+        },
+        "swupdateaction" : {
+          "description": "Scheduled action to do a software update",
+          "maxLength": 64,
+          "type": "string",
+          "enum": [
+              "idle",
+              "isac",
+              "isvv",
+              "upgrade"
+            ]
+        },
+        "swupdatestate" : {
+          "description": "State of the software update",
+          "readOnly": true,
+          "type": "string",
+          "enum": [
+              "idle",
+              "nsa",
+              "svv",
+              "sva",
+              "upgrading"
+            ]
+        },        
+        "result" : {
+          "description": "Result of the software update, list of result codes",
+          "type": "integer"
+        },
+        "lastupdate" : {
+          "description": "Time of the last software update (in RFC3339 format), Initial set on date of manufacturing",
+          "maxLength": 64,
+          "type": "string",
+          "format": "date-time"
+        },
+        "signage" : {
```

Comment:
The Property Name per the CR is "signed", not "signage".

Reply:
This is the comment I just mentioned on the call.

Reply:
fixed